x/ibc/04-channel/types/msgs.go

```diff
@@ -386,7 +386,7 @@ func (msg MsgPacket) GetSigners() []sdk.AccAddress {
 
 // Type implements sdk.Msg
 func (msg MsgPacket) Type() string {
-	return "ics04/opaque"
+	return "ics04/recv"
```

Comment:
do we need the `ics04/` here?

Reply:
it would be nice to have `recv_packet` as it matches the attribute, e.g. "recv_packet.packet_src_channel":["ibconexfer"],
```suggestion
	return "recv_packet"
```

Reply:
also change `ics04/timeout` to `timeout_packet`? In general `ics04` could be inferred as `module` gives the context, `ibc_channel` in this case.

Reply:
I agree. I will update these